src/main/java/uk/gov/hmcts/darts/common/repository/AutomatedTaskRepository.java

```diff
@@ -4,7 +4,10 @@
 import org.springframework.stereotype.Repository;
 import uk.gov.hmcts.darts.common.entity.AutomatedTaskEntity;
 
+import java.util.Optional;
+
 @Repository
 public interface AutomatedTaskRepository extends JpaRepository<AutomatedTaskEntity, Integer> {
 
+    Optional<AutomatedTaskEntity> findByTaskName(String taskName);
```

Comment:
Please can you check you are happy with definition we have in the modern schema / base script?
`TABLE automated_task` it feels like task_name should be unique
If so, please can you involve Judith W. to help make that change to the v41+ schema

Reply:
Changes to the database will be done in a separate ticket.